### PR TITLE
AND | OR support for container search.

### PR DESCRIPTION
**Pull request by Ascynx**
Yeah that's about it.

it's based on [NEUManager#search(String, boolean)](https://github.com/NotEnoughUpdates/NotEnoughUpdates/blob/1e71e3e515fc17034e2e73a9b5e6012637204453/src/main/java/io/github/moulberry/notenoughupdates/NEUManager.java#L546)

### CODE REVIEW

File: src/main/java/io/github/moulberry/notenoughupdates/NEUManager.java [hunk 1]
```diff
@@ -430,6 +430,33 @@ private String searchDebug(String[] searchArray, ArrayList<DebugMatch> debugMatc
 		return debugBuilder.toString();
 	}
 
+	/**
+	 * SearchString but with AND | OR support
+	 */
+
+	public boolean MultiSearchString(String match, String query) {
```
Comment (hannibal002) [inline src/main/java/io/github/moulberry/notenoughupdates/NEUManager.java:437]: mULTIsEARCHsTRING

**Review by hannibal002 — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/moulberry/notenoughupdates/NEUManager.java (hunk 1 above)
Comment (Ascynx) [reply src/main/java/io/github/moulberry/notenoughupdates/NEUManager.java:437]: first thing I thought of, if you got a better idea, then please suggest so.

Comment (jani270) [reply src/main/java/io/github/moulberry/notenoughupdates/NEUManager.java:437]: multiSearchString 🤓

Comment (Ascynx) [reply src/main/java/io/github/moulberry/notenoughupdates/NEUManager.java:437]: > multiSearchString nerd_face

hmmm, I don't think I can put a space in a method name :trollface:

Comment (hannibal002) [reply src/main/java/io/github/moulberry/notenoughupdates/NEUManager.java:437]: I was thinking about `multiSeachString`. It took me 5 seconds to realize you dont have an inner class with constructor there but a normal method.
Please use the camel case convention. See https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

Comment (Ascynx) [reply src/main/java/io/github/moulberry/notenoughupdates/NEUManager.java:437]: ah yeah, fair, don't know how I forgot that

**Review by hannibal002 — APPROVED** [review]

**Review by IRONM00N — APPROVED** [review]
finally

**Review by NopoTheGamer — APPROVED** [review]

**Review by IRONM00N — APPROVED** [review]

**Review by hannibal002 — APPROVED** [review]

**Pull request merged**